src/js/breadcrumbs/breadcrumbs-view.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * A Backbone view that renders breadcrumbs-type tiered navigation.
+ *
+ * Initialize the view by passing in the following attributes:
+ *
+ *~~~ javascript
+ * var view = new DropdownMenuView({
+ *     el: $('selector for element that will contain breadcrumbs'),
+ *     model: new BreadcrumbsModel({
+ *         breadcrumbs: [{url: '/', title: 'Overview'}]
+ *     }),
+ *     events: {
+ *         'click nav.breadcrumbs a.nav-item': function (event) {
+ *             event.preventDefault();
+ *             window.location = $(event.currentTarget).attr('href');
+ *         }
+ *     }
+ * });
+ *~~~
+ * @module BreadcrumbsView
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', '../utils/html-utils.js', 'text!./breadcrumbs.underscore'],
```

Comment:
@bjacobel I don't know where my comment went, but I had trouble using .. paths when consuming the UI Toolkit in edx-platform. I think this needs to be ``edx-ui-toolkit/js/utils/html-utils.js``. You can see in html-utils itself that we do this to access string-utils.